flink-connector-aws/flink-connector-dynamodb/src/main/java/org/apache/flink/connector/dynamodb/source/enumerator/DynamoDbStreamsSourceEnumeratorStateSerializer.java

```diff
@@ -29,15 +29,19 @@
 import java.io.DataInputStream;
 import java.io.DataOutputStream;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.HashSet;
+import java.util.List;
 import java.util.Set;
 
 /** Used to serialize and deserialize the {@link DynamoDbStreamsSourceEnumeratorState}. */
 @Internal
 public class DynamoDbStreamsSourceEnumeratorStateSerializer
         implements SimpleVersionedSerializer<DynamoDbStreamsSourceEnumeratorState> {
 
-    private static final int CURRENT_VERSION = 0;
+    private static final Set<Integer> COMPATIBLE_VERSIONS = new HashSet<>(Arrays.asList(0, 1));
```

Comment:
we have not actually released this connector - so we don't need to up the serializer version even though we are making backwards incompatible changes. This is different from the KDS connector. Can we keep the `CURRENT_VERSION` as 0?